app/Settings/BaseSettings.php

```diff
@@ -120,6 +120,9 @@ public function field_callback_password( $args ) {
 		echo $html;
 	}
 
+	public function field_callback_heading( $args ) {
+	}
+
```

Comment:
@zildjianjax is this needed. Please remove because there is no content on this method.

Reply:
i see make sense